extra-libs/libfido2/autobuild/defines

```diff
@@ -0,0 +1,6 @@
+PKGNAME=libfido2
+PKGSEC=libs
+PKGDES="A library which providing functionality for FIDO 2.0 over USB communications."
```

Comment:
No period point in PKGDES, please.

Reply:
I'd suggest changing this to "A library  which provides FIDO 2.0 functionality over USB"